MREUnityRuntime/MREUnityRuntimeLib/API/MREApi.cs

```diff
@@ -30,6 +31,7 @@ public static class MREAPI
         /// <param name="gltfImporterFactory">The glTF loader factory. Uses default GLTFSceneImporter if omitted.</param>
         /// <param name="logger">The logger to be used by the MRE SDK.</param>
         public static void InitializeAPI(
+            UnityEngine.Material defaultMaterial,
             IBehaviorFactory behaviorFactory = null,
             ITextFactory textFactory = null,
             IPrimitiveFactory primitiveFactory = null,
```

Comment:
>IPrimitiveFactory primitiveFactory = null, [](start = 12, length = 42)

Do you need to remove primitiveFactory here?

Reply:
I'm leaving this in so that host apps can supply their own implementations of the PrimitiveFactory. They don't have to now though.